### PR TITLE
Ignore unnecessary resources

### PR DESCRIPTION
**Pull request by RobinKnipe**
Don't commit IntelliJ clutter.
Don't bundle anything except the `dist/` folder and the `README.md` into the NPM release.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-react-components",
-  "version": "0.12.1",
+  "version": "0.13.0",
```
Comment (daniel-ac-martin) [inline package.json:3]: Is this a precaution in case something breaks? (Or a correction?)
Seems like this should be a patch.

File: .npmignore [hunk 2]
```diff
@@ -1,2 +1,8 @@
 src/
-*.jsx
+test/
+./*.js
+./*.jsx
+./*.json
+.babelrc
+.idea/
+.floo*
```
Comment (daniel-ac-martin) [inline .npmignore:8]: Elsewhere, I've replaced `.npmignore` in favour of a whitelist approach; providing the `files` property inside the `package.json`. i wonder if that might be a better approach.

Comment (daniel-ac-martin) [reply .npmignore:8]: Have a read of https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d and see what you think.

**Comment by daniel-ac-martin** [comment]
This LGTM but it's been a long time since I've worked on this

@aarongill1: Do you have any thoughts on this? Do we need anything outside of `dist/`?

**Comment by aarongill1** [comment]
> This LGTM but it's been a long time since I've worked on this
> 
> @aarongill1: Do you have any thoughts on this? Do we need anything outside of `dist/`?

Nope, dist/ is all that's required for the package.

File: .npmignore (hunk 2 above)
Comment (RobinKnipe) [reply .npmignore]: Ah yes, brilliant, that's exactly what I wanted to do, thanks @daniel-ac-martin ! Fixed

File: package.json (hunk 1 above)
Comment (RobinKnipe) [reply package.json:3]: Yup... the first one. Agreed it _probably_ should be a patch, but as it _may_ break something, not a patch.

**Pull request merged**